src/main/java/com/synopsys/integration/alert/channel/email/EmailMessagingService.java

```diff
@@ -200,8 +200,8 @@ private void sendAuthenticated(final EmailProperties emailProperties, final Mess
     }
 
     private String generateContentId(final String value) {
-        final String cid = value.replaceAll("[^A-Za-z0-9]", "bd").trim() + "@blackducksoftware.com";
-        return cid;
+        // TODO should this be changed to '@synopsys.com'?
+        return value.replaceAll("[^A-Za-z0-9]", "bd").trim() + "@blackducksoftware.com";
```

Comment:
I believe this is just a unique identifier in the content of the email message so changing to synopsys.com should not cause any issues here.